sigh/lib/sigh/resign.rb

```diff
@@ -122,6 +122,7 @@ def installed_identies
       available.split("\n").each do |current|
         begin
           (ids << current.match(/.*\"(.*)\"/)[1])
+          (ids << current.match(/[a-zA-Z0-9]{40}/).to_s)
```

Comment:
Can you add this to the previous `match` so that the SHA-1 is in the same item as the name of the profile it belongs to? Then when `installed_identies` is called you'll see the appropriate number of rows instead of 2 rows for each profile (1 for the profile name and 1 for the SHA-1)